subgraph/subgraph.yaml

```diff
@@ -7,9 +7,10 @@ dataSources:
     name: Unlock
     source:
       abi: Unlock
+      address: "0x2279B7A0a67DB372996a5FaB50D91eAA73d2eBe6"
     mapping:
       kind: ethereum/events
-      apiVersion: 0.0.6
+      apiVersion: 0.0.7
```

Comment:
This was required to get the `receipts` objects!

Reply:
good find !